src/lib.rs

```diff
@@ -215,6 +224,31 @@ pub fn parse(script: &str) -> Result<Vec<Record>, Error> {
     Ok(records)
 }
 
+/// Parse a sqllogictest file and link all included scripts together.
+#[doc(hidden)]
+pub fn parse_file(filename: &str) -> Result<Vec<Record>, Error> {
+    parse_file_inner(Path::new(filename))
```

Comment:
The path should be relative to source file which contains `include` directive.

Reply:
But we need an entry..